packages/lit-html/package.json

```diff
@@ -23,11 +23,18 @@
   ],
   "scripts": {
     "build": "tsc",
+    "test": "wtr test/**/*_test.js --node-resolve",
+    "format": "prettier src/* --write",
     "check-version": "node check-version-tracker.js"
   },
   "author": "The Polymer Authors",
   "dependencies": {},
   "devDependencies": {
+    "@bundled-es-modules/chai": "^4.2.2",
```

Comment:
As @LarsDenBakker mentioned in the [Slack](https://polymer.slack.com/archives/C03PF4L4L/p1598336902325200), it is now recommended to use [@esm-bundle/chai](https://www.npmjs.com/package/@esm-bundle/chai) which auto updates whenever there is an update from chai.